.gitignore

```diff
@@ -18,3 +18,5 @@ coverage.html
 build/*/tmp/*
 contrib/build/*/tmp/*
 .pkg
+.kube
+.var
```

Comment:
not sure what'd you'd expect to see aside from "we don't want to check these in"